data/common.yaml

```diff
@@ -1 +1,16 @@
---- {}
+---
+
+profile_gpu::dcgm::install::bind_mount_install:        # See reference.md for when this is needed, set to undef on purpose
```

Comment:
I think it might make sense to set this more clearly. I didn't realize you could leave it blank/empty to be undefined. I thought you could use `undef` or something similar, which to me seems more obvious.

Reply:
Good point. Setting it to `undef` made it fail but with a `expected boolean but got string` error. Tried `null` which gave a `expected bolean but got undef` error, which lines up better so I used that.